client/rest-high-level/src/main/java/org/elasticsearch/client/ml/job/process/ModelSizeStats.java

```diff
@@ -47,6 +47,8 @@
      * Field Names
      */
     public static final ParseField MODEL_BYTES_FIELD = new ParseField("model_bytes");
+    public static final ParseField MODEL_BYTES_EXCEEDED_FIELD = new ParseField("model_bytes_exceeded");
+    public static final ParseField MODEL_BYTES_MEMORY_LIMIT_FIELD = new ParseField("model_bytes_memory_limit");
```

Comment:
This is copied from `analysis_limits.model_memory_limit`, so I think it would best to call this `model_memory_limit`.  That will make it clearer that it refers to the same thing as the `model_memory_limit` from the job config.

Reply:
On reflection scrub the above comment. `analysis_limits.model_memory_limit` is megabytes whereas this one is bytes so it makes sense to have a slightly different name.